plugins/scss/index.js

```diff
@@ -156,9 +156,17 @@ const addScssRequires = function(pack, dir) {
 
 const writer = function(thisPackage, outputDir) {
   if (scssPaths.length || scssEntries.length) {
-    var options = {
-      'include-path': scssPaths
-    };
+    var options = {};
+    if ((thisPackage.build && thisPackage.build.scssCompiler === 'sass') ||
```

Comment:
With other upgrades like webpack, we left the original output intact (ie, `.build/gcc-args.json`) and created a new output file for the upgrade (`gcc-webpack.json`). Instead of changing the output of `node-sass-args`, it seems simpler to leave that alone and also write a `sass-args` file. Then the project can choose which to use in its build scripts, and support both if it needs to.

Reply:
Is this change a nice-to-have? Or is it necessary? 
The `writer` function in this plugin would require a fair amount of change to make that happen in a clean way.

Reply:
We chatted about this. There is probably a better way to do this (e.g. add another writer that is referenced by this one or something like that)